.evg.yml

```diff
@@ -175,7 +175,7 @@ buildvariants:
   - name: ubuntu1804
     display_name: Ubuntu 18.04
     expansions:
-      build_env: "GOROOT=/opt/golang/go1.16 PATH=/opt/golang/go1.16/bin:$PATH"
+      build_env: "GOROOT=/opt/golang/go1.18 PATH=/opt/golang/go1.18/bin:$PATH"
```

Comment:
Why do we want to upgrade go version? Is it just to stay current?

Reply:
Mostly because I wanted to use UnixMilli... I can rewrite the code without it though.

Reply:
It _is_ nice to stay current and it wasn't too hard. I only ended up having to also update the linter and the few places where we use conditional build instructions.

Reply:
Copying over from Slack: I can't find where the code uses `UnixMilli`. If it does I'd think we definitely need to increment the go directive to get the build to work.

Reply:
It was one of the unused functions that we deleted.

I'm going to revert the upgrade to 1.8 for now because we don't need it and it should be a separate PR.